starter-project-web-react/store/slices/blogs.js

```diff
@@ -37,8 +37,8 @@ export default slice.reducer
 
 export const addBlog = (post) => (dispatch, getState) => {
   // when apiCallBegan dispached the middleware(api middleware) we have configured in the configureStore will handle it first
-  const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFiZW5AZ21haWwuY29tIiwic3ViIjoiNjI5NjE5OWE5NWI3ZGIyMmRlMzY1MDdmIiwiaWF0IjoxNjU0MDk1NzU5LCJleHAiOjE2NTQwOTkzNTl9.-WPvKbZ1LDNkhl_aWVW4y21G-UD4UGobXMe3tVutA-M"
-     const data = {title:post.title, content: post.content,description: post.content, }
+  const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFiZW5AZ21haWwuY29tIiwic3ViIjoiNjI5NjE5OWE5NWI3ZGIyMmRlMzY1MDdmIiwiaWF0IjoxNjU0NDkwMzYyLCJleHAiOjE2NTQ0OTM5NjJ9.RI5Wamlrt12FDlj8bNhidgcdSIJH2DhL5VcuHzs7EBM"
```

Comment:
better not to hard code token
if it is something that can be found by logging in, better to fetch it from the session
if it is something constant, better to save it in a .env file and keep that in .gitignore. You can have a .env.example file where you can have a template of what varibles are used from the .env file. this can be kept out of the .gitignore so that people would know how to create their own .env files

Reply:
I will change it I just told minte to prepare the function for that until then yahibibi we don't have a choice